CHANGELOG.md

```diff
@@ -0,0 +1,61 @@
+# Changelog QualityOnDemand
+
+## Content
+
+- [v0.8.0](#v080)
+- [v0.1.0 - Initial contribution](#v010---initial-contribution)
+
+Version number 0.2.x to 0.7.x were intentionally not used to avoid conflicts with local implementations.
+
+**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
+
+
+# v0.8.0
+
+**This is the first alpha version of the Quality-On-Demand (QoD) API.** 
+- API [definition](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.0/code/API_definitions)
+- API [documentation](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.0/documentation/API_documentation)
+
+## Please note:
+
+- This is an alpha version, it should be considered as a draft.
+- There are bug fixes to be expected and incompatible changes in upcoming versions. 
+- The release is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
+- Version numbers 0.2.x to 0.7.x where used in privat versions during the development of the API and are here not used to avoid conflicts with local implementations.
```

Comment:
s/where used in privat /were used in private /g

Reply:
Done in [19eba91](https://github.com/camaraproject/QualityOnDemand/pull/99/commits/19eba91458b220e458c2604b4a0a5fa03a50519d)